platform/view/services/db/keys/utils.go

```diff
@@ -14,8 +14,8 @@ import (
 )
 
 var (
-	nsRegexp  = regexp.MustCompile("^[a-zA-Z0-9._-]{1,20}$")
-	keyRegexp = regexp.MustCompile("^[a-zA-Z0-9._\u0000=" + string(utf8.MaxRune) + "+/-]{1,200}$")
+	nsRegexp  = regexp.MustCompile("^[a-zA-Z0-9._-]{1,50}$")
```

Comment:
why still limiting the namespace to 50 char?

Reply:
Good point, I don't think there is a rule here. The limit there is not hurting for now :)